### ISSUE TITLE
Bug? (mouseover) on a component

### ISSUE THREAD

**Issue by MichaelRFairhurst** [dart-lang/angular_analyzer_plugin#101]
Need info if this is a bug (in other words, step one is to play around with this in a live angular app).

Seeing a project where (mouseover) is bound on a component:

```
   <custom-list-component (mouseover)="action">
   ...
```

I have a feeling this _does_ work, and that it binds to the outermost tag of the component. Ie,

```
@Component(template: "<div>...</div>", ...)
class MyComponent {
```

would have all the dom events that `div` does. If so, this could probably recurse, if the first element of `MyComponent` is a different component which has a template that starts with an html tag or a component that..... etc.

That said, this _might_ just be a bug in the codebase we were analyzing, that leads to a noop.

**Comment by matanlurey** [comment]
Native events get caught up the DOM tree, like normal DOM would.

i.e.

```html
<div (click)="doSomething()">
  <button>Click me, it will still trigger doSomething</button>
</div>
```

Only native events have this property, though.

**Comment by MichaelRFairhurst** [comment]
Great!

Looks like the DOM API already accounts for this; all bubbling events are present on the single type `HtmlElement`, since any element can bubble events from their children. We should be able to just make every component inherit all the outputs of HtmlElement from `dart:html` from the analyzer's perspective to solve this.

**Comment by MichaelRFairhurst** [comment]
Merged PR #106
